### ISSUE TITLE
Let "docker compose" start "docker-compose" and pass all additional arguments.

### ISSUE THREAD

**Issue by nudgegoonies** [docker/cli#2074]
FEATURE_REQUEST:
Let "docker compose" start "docker-compose" and pass all additional arguments.

People are so used to docker and docker-compose commands and often have to use docker and docker-compose in parallel. This leads to lots of these annoying errors:
```
docker: 'compose' is not a docker command.
See 'docker --help'
```

**Comment by PMExtra** [comment]
I can't understand why does `docker stack` is a component of cli, but `docker compose` not.
I know there are some problems left over by history. But now, [docker/compose](https://github.com/docker/compose) is already a project that maintained by docker official, so I think it could be merged into cli.

**Comment by jamshid** [comment]
IDK but I think this is the direction Docker is going in, replacing the separate `docker-compose` python-based tool with `docker compose` built into the regular docker cli. 
See the beta project https://github.com/docker/compose-cli. It's coming along nicely, becoming a very compatible replacement for `docker-compose`.
It's available in the docker tool on Docker for Desktop but Linux users (even those on latest 20.10.6) have to install a separate `docker` binary at the moment.

**Comment by ndeloof** [comment]
Things actually happened the other way:
`docker compose` is now a CLI subcommand and [Compose v2](https://github.com/docker/compose-cli) will eventually replace docker-compose. In the meantime, Docker Desktop includes a transition script to translate calls to `docker-compose` into `docker compose ...`
